### ISSUE TITLE
NTR: Bronchiolectasis

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#4707]
**Preferred term label:**
Bronchiolectasis

**Synonyms**
Bronchiolectasia

**Definition (free text, please give PubMed ID)**
A chronic lung disease characterized by abnormal dilatation or widening of the bronchi and mild to moderate airway obstruction, which frequently co-occurs with long-term inflammation, airway infection, and bronchial lesions ([PMID:20037680](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2793069/), [PMID:14256715](https://www.ncbi.nlm.nih.gov/pubmed/14256715)). Clinical manifestations include chronic cough, ineffective clearance of mucus, chronic or recurrent infection, weight loss, fatigue, and in very rare cases, hemotysis ([PMID:26024063](https://www.ncbi.nlm.nih.gov/pubmed/26024063)).

**Parent term (use hpo.jax.org/app)**
HP:0025426 Abnormal bronchus morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cystic fibrosis [OMIM:606963](https://www.omim.org/entry/606963?search=COPD&highlight=copd) ([PMID:28546748](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5436792/))

**Your nano-attribution (ORCID)**
ORCID: [0000-0002-8169-9049](https://orcid.org/0000-0002-8169-9049)

**Comment by pnrobinson** [comment]
I think the definition of the phenotype should be: saccular dilatation of the terminal bronchioles. The disease definition mentioned above might include Bronchiolectasis as a component

**Comment by callahantiff** [comment]
OK, I can see how that would be more specific. Any feedback on creating good definitions is appreciated; I am always trying to find ways to create less work for you :D.

**Comment by nicolevasilevsky** [comment]
I'll add this
